source.js

```diff
@@ -113,7 +118,8 @@
 
 				var code = this.source.$textarea.val();
 
-				code = this.paragraphize.load(code);
+				// Disable to workaround bug in paragraphize, which will keep adding <p> wrapper for some top level element such as <a>
+				// code = this.paragraphize.load(code);
```

Comment:
Are you sure there are no bad side effects from disabling this? I think we need to identify the use case for which this was first enabled, and verify that this isn't a problem.

Reply:
Sorry about that, I actually didn't write this part of the code it was brought in from a different commit that was from a while ago but for some reason became apart of my branch